sw/applications/src/layers/batchnorm_layer.c

```diff
@@ -0,0 +1,136 @@
+// Copyright 2020 ETH Zurich and University of Bologna.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "batchnorm_layer.h"
+
+#include "batchnorm.h"
+#include "layer.h"
+#include "snrt.h"
+
+void batchnorm_layer(layer l) {
```

Comment:
the `layer` struct is passed as argument. This is **>200 bytes** of data passed through stack. passing a pointer would make it faster, more energy efficient and reduces the chance of stack-overflow.